### ISSUE TITLE
Docker duplicates the complete Application Event Log

### ISSUE THREAD

**Issue by Hickstone** [docker/for-win#6258]
- [x] I have tried with the latest version of my channel (Stable)
  - [ ] I have uploaded Diagnostics (I won't upload as it may contain sensitive data)

### Expected behavior
The Files %LOCALAPPDATA%\Docker\log.txt and %PROGRAMDATA%\DockerDesktop\service.txt contain only docker relevant information.

### Actual behavior
The logs contain ALL log entries from the Windows Application Event Log.
The files will also contain the same content, but I think that may be to enable multi user diagnostics. (I'm the only docker user on my computer)

### Information
  - Is it reproducible? Yes.
  - Is the problem new? Don't know. I found it while searching for another problem.
  - Did the problem appear with an update? Same as above
  - Windows Version: Windows 10 Enterprise 1809 (17763.1039)
  - Docker Desktop Version: 2.2.0.5
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: No.

### Steps to reproduce the behavior
Start Powershell with Admin rights and type the following:
```powershell
# Only if the source "foobar" doesn't exist
[System.Diagnostics.Eventlog]::CreateEventSource("foobar", "Application")
Write-EventLog -LogName Application -Source foobar -EventID 1234 -Message "Hello FooBar!"
```
The created log message will appear in the log.txt and service.txt desipite it's relevance to Docker.

**Comment by mat007** [comment]
Thanks for the report.
There must be something else going on, as I can’t reproduce it using the same Docker Desktop version and the PS commands you provided.
🤔

**Comment by mat007** [comment]
Oh I just realized I wasn’t in Windows containers mode…
I can confirm I am able to reproduce the issue, thanks!

**Comment by mat007** [comment]
This has now been fixed and will be available in the next release.
Thanks!

**Comment by stephen-turner** [comment]
Fixed in 2.3, closing this ticket.
